src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -5140,3 +5190,244 @@ class CreateExternalDataSourceStatementSegment(BaseSegment):
             ),
         ),
     )
+
+
+class ExternalFileFormatDelimitedTextFormatOptionClause(BaseSegment):
+    """`CREATE EXTERNAL FILE FORMAT` Delimited text `FORMAT_OPTIONS` clause."""
+
+    type = "external_file_delimited_text_format_options_clause"
+
+    match_grammar = OneOf(
+        Sequence(
```

Comment:
Most of these can be condensed into one Sequence. The ones that equal quotedliteral can have the options wrapped in a OneOf()
OneOf("FIELD_TERMINATOR", "STRING_DELIMITER") etc